### PR TITLE
Added extra validations & updated README

### PR DESCRIPTION
**Pull request by stevelawsonspi**

### CODE REVIEW

File: test/models/location_test.rb [hunk 1]
```diff
@@ -9,6 +9,18 @@ def setup
     assert @location.valid?
   end
   
+  test "rejects blank name" do
+    @location.name = ''
+    refute       @location.valid?
```
Comment (wizardofosmium) [inline test/models/location_test.rb]: I would recommend adding a message to help with debugging: e.g.
```ruby
    refute       @location.valid?, 'a blank name should not be valid'
```

Comment (stevelawsonspi) [reply test/models/location_test.rb]: cool, thanks.  Showing my lack of minitest experience :)  do you use minitest or rspec at Shippit?

Comment (wizardofosmium) [reply test/models/location_test.rb]: We use RSpec at Shippit.

Comment (stevelawsonspi) [reply test/models/location_test.rb]: 👍

**Pull request merged**